ClassLibrary/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ClassLibrary
+{
+    public class Employee
+    {
+        private string firstName;
+        private string lastName;
+        private int age;
+        private int experienceYears;
+        private string position = "Intern";
+
+        public string getPosition
+        {
+            get { return position; }
+            set { position = value; }
+        }
+        public Employee()
+        {
+            firstName = "New";
+            lastName = "Employee";
+        }
+        public Employee(string firstName, string lastName, int age)
+        {
+            this.firstName = firstName;
+            this.lastName = lastName;
+            this.age = age;
+        }
```

Comment:
add 1 blank line everywhere between methods to improve readability

Reply:
done